spec/test_app/app/controllers/basechurch/v1/posts_controller_spec.rb

```diff
@@ -37,23 +44,34 @@
   #  - response: The response of an action
   #  - expected_post: The expected post
   shared_examples_for 'a response containing a post' do
-    let(:body) { JSON.parse(response.body) }
+    let(:data) { JSON.parse(response.body)["data"] }
+    let(:attributes) { data["attributes"] }
+    let(:group_data) { data["relationships"]["group"]["data"] }
+    let(:author_data) { data["relationships"]["author"]["data"] }
+    let(:editor_data) { data["relationships"]["editor"]["data"] }
+
     it 'returns a post' do
-      expect(body['posts']['id']).to eq(expected_post.id.to_s)
-      expect(body['posts']['content']).to eq(expected_post.content)
-      expect(body['posts']['slug']).to eq(expected_post.slug)
-      expect(body['posts']['tags']).to eq(expected_post.tag_list)
-      expect(body['posts']['title']).to eq(expected_post.title)
-      expect(body['posts']['publishedAt']).to eq(expected_post.published_at.to_time.localtime('+00:00').iso8601)
-      expect(body['posts']['links']['group']).to eq(expected_post.group_id.to_s)
-      expect(body['posts']['links']['author']).to eq(expected_post.author_id.to_s)
+      expect(data['id']).to eq expected_post.id.to_s
+
+      expect(attributes['content']).to eq expected_post.content
+      expect(attributes['slug']).to eq expected_post.slug
+      expect(attributes['tags']).to eq expected_post.tag_list
+      expect(attributes['title']).to eq expected_post.title
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.